src/_js/map.js

```diff
@@ -181,6 +181,43 @@
 			}
 		});
 	};
+	TibiaMap.prototype._loadMarkers = function () {
+		const _this = this;
+		const icons = []
+		const symbols = ["!", "$", "?", "bag", "checkmark", "cross", "crossmark", "down", "flag", "lock", "mouth", "red down", "red left", "red right", "red up", "skull", "spear", "star", "sword", "up"];
+		symbols.forEach(s => {
+			icons[s] = L.icon({iconSize: [11, 11], iconUrl: s.replace("!", "exclamation").replace("$", "dollar").replace("?", "question").replace(" ", "_") + ".png"});
+		});
+
+		const xhr = new XMLHttpRequest();
+		xhr.open('GET', 'https://tibiamaps.github.io/tibia-map-data/markers.json', true);
```

Comment:
`async=true` is the implicit default, so no need to specify it.

```suggestion
		xhr.open('GET', 'https://tibiamaps.github.io/tibia-map-data/markers.json');
```

We could also use `fetch` these days, by the way. The only reason we use XHR in the old code is because it was written before `fetch` was widely available in browsers.

Reply:
I suggest keeping XHR for consistency now, we can replace with `fetch` in a follow up patch